src/components/Reviews/Reviews.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import {
+  Divider, List, ListItem, ListSubheader, Grid
+} from "@mui/material";
+import reviews from "../TestData/reviews.json";
+import albums from "../TestData/profilealbums.json";
+import Typography from "@mui/material/Typography";
+import "../Reused/reused.css";
+import ReviewEntry from "./ReviewEntry";
+import ImageText from "../Reused/ImageText";
+import Rating from "@mui/material/Rating";
+import Box from "@mui/material/Box";
+
+const Reviews = ({id, idType}) => {
+
+  let displayReviews;
+  if (idType === "user") {
+    displayReviews = reviews.filter(review => review.user_id == id);
+  } else {
+    displayReviews = reviews.filter(review => review.album_id == id);
+  }
```

Comment:
We will be able to query out backend/db for reviews based on a user_id or a album_id, so the results we get will already be filtered for us.  We can leave for now, but likely we will not need to handle this here in the future.